src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -126,7 +200,7 @@ HttpStatusCode mongoUpdateContextSubscription
   else
   {
     newSub.append(CSUB_EXPIRATION, expiration);
-    LM_T(LmtMongo, ("New subscription expiration: %l", expiration));
+    LM_T(LmtMongo, ("New subscription expiration: %" PRId64, expiration));
```

Comment:
a "constant" (macro for a string literal) for the right format specifier for int64_t